client/src/component/customer/ReviewPage.js

```diff
@@ -0,0 +1,119 @@
+import { useState, useEffect } from "react";
+
+import { getReviews } from "../../api/googleApi.js";
+
+import { Box, Text } from "@chakra-ui/react";
+
+import { Avatar, Stack } from "@chakra-ui/react";
+import { StarIcon } from "@chakra-ui/icons";
+
+function ReviewPage() {
+  let [apiData, setApiData] = useState(null);
+
+  const getApiData = async () => {
+    setApiData(await getReviews());
+  };
+
+  useEffect(() => {
+    getApiData();
+  }, []);
+
+  return apiData ? (
+    <>
+      <Box
+        mt="1"
+        fontWeight="semibold"
+        as="h1"
+        fontSize="4xl"
+        // lineHeight="tight"
+        noOfLines={1}
+      >
+        Hotel GURUPRASAD
+      </Box>
+      <Box
+        color="gray.500"
+        fontWeight="semibold"
+        letterSpacing="wide"
+        fontSize="md"
+      >
+        Ujire, Karnataka 574240, India
+      </Box>
+      <Box display="flex" alignItems="baseline">
+        <Box display="flex" mt="2" alignItems="center">
+          {Array(5)
+            .fill("")
+            .map((_, i) => (
+              <StarIcon
+                key={i}
+                color={i < apiData.result.rating ? "orange.500" : "gray.300"}
+              />
+            ))}
+        </Box>
+        <Box as="span" ml="2" color="gray.600" fontSize="sm">
+          {apiData.result.user_ratings_total} reviews
+        </Box>
+      </Box>
+      {/* Mapping*/}
+
+      {apiData.result.reviews.map((author) => {
```

Comment:
You're iterating over the reviews here so it would make more sense to call this `review`.

```suggestion
      {apiData.result.reviews.map((review) => {
```